conda_pack/prefixes.py

```diff
@@ -72,6 +75,12 @@ def update_prefix(path, new_prefix, placeholder, mode='text'):
         if data != original_data:
             fh.write(data)
             fh.truncate()
+            file_changed = True
+
+    if file_changed and platform.system() == "Darwin" and platform.machine() == "arm64":
+        subprocess.run(
+            ["/usr/bin/codesign", "-s", "-", "-f", path], capture_output=True
+        )
```

Comment:
Should the output be logged if the command failed? Something like:

```suggestion
        process = subprocess.run(
            ["/usr/bin/codesign", "-s", "-", "-f", path], capture_output=True, text=True,
        )
        if process.returncode:
            log.debug("codesign failed for '%s'.\nstdout:\n%s\nstderr:\n%s", path, process.stdout, process.stderr)
```

Reply:
I'm keeping the same behaviour here as we do in conda: https://github.com/conda/conda/blob/a3648d7b58175fd505b3176e45e3485c6d2ea571/conda/core/portability.py#L87

I can add that logging but it will probably never pop to the user with being on debug level.